pennylane/qchem/molecule.py

```diff
@@ -193,8 +193,8 @@ def orbital(x, y, z):
                 array[float]: value of a basis function
             """
             c = ((x - r[0]) ** lx) * ((y - r[1]) ** ly) * ((z - r[2]) ** lz)
-            e = [np.exp(-a * ((x - r[0]) ** 2 + (y - r[1]) ** 2 + (z - r[2]) ** 2)) for a in alpha]
-            return c * np.dot(coeff, e)
+            e = [pnp.exp(-a * ((x - r[0]) ** 2 + (y - r[1]) ** 2 + (z - r[2]) ** 2)) for a in alpha]
+            return c * pnp.dot(coeff, e)
```

Comment:
Could we use `qml.math` here?

Reply:
Yes! But I would suggest that using `qml.math` in different parts of the `qchem` module could be done separately.